content/docs/tasks/telemetry/access-log/index.md

```diff
@@ -0,0 +1,92 @@
+---
+title: Envoy's Access Logging
```

Comment:
Suggest: Getting Envoy's Access Logs

Reply:
Accepted in https://github.com/istio/istio.io/pull/3531/commits/221b13d437f25bf9608108fbf830eb1831c9fcfc